manifests/init.pp

```diff
@@ -174,10 +174,8 @@
       force   => true,
       recurse => true;
   } ~> service { 'nftables':
-    ensure     => running,
-    enable     => true,
-    hasrestart => true,
-    restart    => '/usr/bin/systemctl reload nftables',
```

Comment:
Switching to 

```
systemctl restart -- nftables
```
That is 'stop' and 'start'  which

```
ExecStop=/sbin/nft flush ruleset
ExecStart=/sbin/nft -f /etc/sysconfig/nftables.conf
```
which would definitely be bad.

Reply:
oh right, that would flush the rules.